Tasks/PackerBuildV1/task.json

```diff
@@ -245,6 +245,15 @@
             "defaultValue": "",
             "helpMarkDown": "Provide a name for the output variable which will store generated machine image VHD url for un-managed VM image or image name for managed VM image.",
             "groupName": "Output"
+        },
+        {
+            "name": "imageId",
+            "type": "string",
+            "label": "Azure Resource Id",
```

Comment:
@pulkitaggarwl @azooinmyluggage @bishal-pdMSFT 
does the label look okay for this input ?

Reply:
I think so 
![image](https://user-images.githubusercontent.com/37842906/50690620-fc8fc800-0ffb-11e9-9818-a686433ee6ae.png)

![image](https://user-images.githubusercontent.com/37842906/50690637-0dd8d480-0ffc-11e9-9537-69d588e6b160.png)

Reply:
@grayjeremy please add a note that this should only be used when generated image is a Managed Image.